guides/get-started-microservices-on-kubernetes/simple-microservice-database-part1.adoc

```diff
@@ -0,0 +1,413 @@
+----
+
+=== Check the application
+
+Hit the following URLs, using a utility like `curl`:
+
+.Insert some Data into the Database:
+[source,bash]
+----
+$ curl -X POST http://localhost:8080/hello/test-table/insert/somedata1/somedata2
+{"field1":"somedata1","field2":"somedata2","id":1}
+----
+
+.Query the database to show the inserted data:
+[source,bash]
+----
+$ curl http://localhost:8080/hello/test-table/list
+[{"field1":"somedata1","field2":"somedata2","id":1}]
+----
+
+== Docker Image
+
+=== Build the Docker Image
+
+Build the Docker Image with the following command:
+
+[source,bash,subs="normal"]
+----
+$ podman build -t {my-jaxrs-app-db-docker-image-name}:latest .
+STEP 1/3: FROM quay.io/wildfly/wildfly-runtime:latest
+STEP 2/3: COPY --chown=jboss:root target/server $JBOSS_HOME
+--> 4609f8ed0c7f
+STEP 3/3: RUN chmod -R ug+rwX $JBOSS_HOME
+COMMIT my-jaxrs-app:latest
+--> db4677f5bf4f
+Successfully tagged localhost/my-jaxrs-app:latest
+db4677f5bf4f471f5624bd63a21fce3d91b7b3b93e985d3e86a8a4b0682d85cd
+----
+
+=== Run the Docker Image
+
+Note that, when running the `{my-jaxrs-app-db-docker-image-name}:latest` Docker Image, we specify some environment variables used by WildFly to connect to the PostgreSQL database:
+
+[source,bash,subs="normal"]
+----
+podman run --rm -p 8080:8080 -p 9990:9990 \
+    -e POSTGRESQL_USER={postgre-sql-user} \
+    -e POSTGRESQL_PASSWORD={postgre-sql-password} \
+    -e POSTGRESQL_HOST=host.containers.internal \
+    -e POSTGRESQL_PORT={postgre-sql-port} \
+    -e POSTGRESQL_DATABASE={postgre-sql-database} \
+    -e POSTGRESQL_JNDI={postgre-sql-jndi} \
+    --name={my-jaxrs-app-db-docker-image-name} \
+    {my-jaxrs-app-db-docker-image-name}:latest
+----
+
+NOTE: Depending on your Podman version, you might have to add the `--network slirp4netns:allow_host_loopback=true` option to Podman which allows to reference the host as `host.containers.internal` from inside the container;
+
+NOTE: if you are using Docker, you need to use `-e POSTGRESQL_HOST=host.docker.internal` instead of `-e POSTGRESQL_HOST=host.containers.internal`
+
+=== Check the application
+
+Just like we did before, hit the following URLs, using a utility like `curl`:
+
+.Insert some Data into the Database:
+[source,bash]
+----
+$ curl -X POST http://localhost:8080/hello/test-table/insert/somedata1/somedata2
+{"field1":"somedata1","field2":"somedata2","id":2}
+----
+
+.Query the database to show the inserted data:
+[source,bash]
+----
+$ curl http://localhost:8080/hello/test-table/list
+[{"field1":"somedata1","field2":"somedata2","id":1},{"field1":"somedata1","field2":"somedata2","id":2}]
+----
+
+NOTE: you might find your data twice if didn't restart the Database
+
+== What's next?
+
+link:simple-microservice-database-part2[{simple-microservice-database-part2}]
+
+[[references]]
+== References
+
+* Source code for this post: {source-code-git-repository}/tree/simple-microservice-database-part1
```

Comment:
I think we are using wildfly-extras to keep all the wildfly.org guides. @jmesnil can you comment about this?

Reply:
@jmesnil can you also create the repo in case? I don't think I have the necessary permissions....

Reply:
@tommaso-borgato Found it here: https://github.com/wildfly-extras/guides, you can fork/clone it this project and create a PR against it adding a new directory for your guide

Reply:
Ahh, you already commented about it at https://github.com/wildfly/wildfly.org/pull/595#issuecomment-2225484845.

Going to resolve this conversation then